src/metabase/analytics/snowplow.clj

```diff
@@ -23,10 +23,12 @@
    (org.apache.http.impl.client HttpClients)
    (org.apache.http.impl.conn PoolingHttpClientConnectionManager)))
 
+(set! *warn-on-reflection* true)
+
```

Comment:
At some point I want to enable the Kondo linter that requires this in every namespace that does Java interop. This way we catch reflection warnings right away. (It only affects the current namespace.)